src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -666,10 +668,13 @@ private static bool IsTypeLessVisibleThan(TypeSymbol type, Symbol sym, ref Compo
             Func<TypeSymbol, T, bool, bool>? typePredicate,
             T arg,
             bool canDigThroughNullable = false,
-            bool useDefaultType = false)
+            bool useDefaultType = false,
+            bool visitCustomModifiers = false)
```

Comment:
>bool visitCustomModifiers = false [](http://example.com/codeflow?start=12&length=33)

It looks like we can take advantage of the ```VisitType``` change in another PR. Could you copy changes made in this file and in SymbolExtensionTests.cs to a separate new PR? So that we can merge them ASAP.

Reply:
Sure, please check #58541